Security/ObjectUserProvider.php

```diff
@@ -56,6 +66,12 @@ public function refreshUser(UserInterface $user)
      */
     public function supportsClass($class)
     {
-        return $this->userRepository->supportsClass($class);
+        try {
```

Comment:
$this->userRepositoryCollection should always have a repository for the correct user type,
if it doesn't exist it throws an RepositoryNotRegisteredException